src/widgets/help_screen.rs

```diff
@@ -41,8 +41,8 @@ pub struct HelpScreen {
 impl HelpScreen {
 	pub fn new(usage_img: Picture) -> HelpScreen {
 		let placement = WidgetPlacement {
-			width: Length::Fixed(0.0 as f32),
-			height: Length::Fixed(0.0 as f32),
+			width: Length::Fixed(0.0_f32),
+			height: Length::Fixed(0.0_f32),
```

Comment:
I think this would look better without marking the type. Like this
```suggestion
			width: Length::Fixed(0.0),
			height: Length::Fixed(0.0),
```

(I have no idea what I was thinking when I originally wrote `0.0 as f32`)

Reply:
Yup, I missed that it's not necessary here. Fixed.